5.5/run-mysqld.sh

```diff
@@ -6,10 +6,14 @@ source $HOME/.bashrc
 set -eu
 
 # Data directory where MySQL database files live. The data subdirectory is here
-# because .bashrc lives in /var/lib/mysql/ and we don't want a volume to
-# override it.
-MYSQL_DATADIR=/var/lib/mysql/data
-MYSQL_DEFAULTS_FILE=/opt/openshift/etc/my.cnf
+# because .bashrc and my.cnf both live in /var/lib/mysql/ and we don't want a
+# volume to override it.
+export MYSQL_DATADIR=/var/lib/mysql/data
+
+# Configuration settings.
+export MYSQL_DEFAULTS_FILE=$HOME/my.cnf
```

Comment:
Yeah, the template has the .template suffix. But the generated file is `$MYSQL_DEFAULTS_FILE` and the variable is used multiple times as argument to different commands later.